src/evernode.c

```diff
@@ -781,11 +801,30 @@ int64_t hook(int64_t reserved)
                 INT64_TO_BUF(ledger_buf, ledger);
                 COPY_BUF(redeem_op, 31, ledger_buf, 0, 8);
 
+                // Set the user address.
+                COPY_BUF(redeem_op, 39, account_field, 0, 20);
+
                 // Set state key with transaction hash(id).
                 REDEEM_OP_KEY(txid);
                 if (state_set(SBUF(redeem_op), SBUF(STP_REDEEM_OP)) < 0)
                     rollback(SBUF("Evernode: Could not set state for redeem_op."), 1);
 
+                // Forward redeem to the host
+                etxn_reserve(1);
+
+                uint8_t data[data_len];
+                COPY_BUF(data, 0, data_ptr, 0, data_len);
+
+                int64_t fee = etxn_fee_base(PREPARE_PAYMENT_REDEEM_SIZE(sizeof(data)));
+
+                uint8_t txn_out[PREPARE_PAYMENT_REDEEM_SIZE(sizeof(data))];
+                PREPARE_PAYMENT_REDEEM(txn_out, MIN_DROPS, fee, issuer_ptr, data, sizeof(data));
+
+                uint8_t emithash[HASH_SIZE];
+                if (emit(SBUF(emithash), SBUF(txn_out)) < 0)
+                    rollback(SBUF("Evernode: Emitting txn failed"), 1);
```

Comment:
Getting following error when performing a redeem (In this case MIN_REDEEM has been set to 1):
```
conf_min_redeem 1
Failed Unknown field in Object t=3 f=13
ROLLBACK RS: 'Evernode: Emitting txn failed' RC: 1
```